Testing/VelaptorTests/Content/TexturePathResolverTests.cs

```diff
@@ -68,22 +69,22 @@ public void ResolveFilePath_WhenContentItemDoesNotExist_ThrowsException()
         // Arrange
         var mockDirectory = new Mock<IDirectory>();
         mockDirectory.Setup(m => m.GetFiles(this.atlasContentDir, "*.png"))
-            .Returns(() =>
-            {
-                return new[]
+            .Returns(
+                new[]
                 {
                     $"{this.baseDir}/other-file-A.png",
-                    $"{this.baseDir}/other-file-B.txt",
-                };
-            });
+                    $"{this.baseDir}/other-file-B.txt"
```

Comment:
Please add a trailing comma to the end of this line.  This should be seen as a warning and is for coding style enforced by the project analyzers.

```suggestion
                    $"{this.baseDir}/other-file-B.txt",
```

**Warning shown as:**
Use trailing comma in multi-line initializers.

> **Note** Please check and see if this warning shows for you in **Visual Studio**.  The coding style enforced is done by the analyzers combined with the _**.editorconfig**_ files and **JetBrains Rider** is setup in more depth then in **Visual Studio**.  I am curious if it shows for you or not.